docs/languages/en/modules/zend.input-filter.intro.rst

```diff
@@ -167,6 +167,197 @@ appropriate object. You may create either ``Input`` or ``InputFilter`` objects i
        ),
    ));
 
+The ``merge()`` method may be used on an ``InputFilterInterface`` in order to add two or more filters to each other, effectively
+allowing you to create chains of filters. This is especially useful in object hierarchies whereby we may define a generic
+set of validation rules on the base object and build these up to more specific rules along the way.
+
+In the example below an ``InputFilter`` is built up for the name property as well as for the email property allowing them to
+be re-used elsewhere. When the ``isValid()`` method is called on the object, all of the merged filters are run against
+the calling object in order to validate the internal properties based on our compound set of filters.
+
+.. code-block:: php
+   :linenos:
+
+        use Zend\InputFilter\InputFilterInterface;
+        use Zend\InputFilter\Factory as InputFactory;
+        use Zend\InputFilter\InputFilter;
+        use Zend\InputFilter\InputFilterAwareInterface;
+        use Zend\InputFilter\InputFilterInterface;
+
+       /**
+        * Filter to ensure a name property is set and > 8 characters
+        */
+        class NameInputFilter extends InputFilter
+        {
+            /** @var InputFactory */
+            protected $inputFactory;
+
+            public function __construct()
+            {
+                $this->inputFactory = new InputFactory();
+                $this->setValidators();
+            }
+
+            /**
+             * Loads the validators
+             */
+            protected function setValidators()
```

Comment:
Please remove this method: we are writing examples, keep them as small as possible.